subprojects/core/src/integTest/groovy/org/gradle/api/PluginApplicationErrorIntegrationTest.groovy

```diff
@@ -41,7 +41,7 @@ apply plugin: 'broken'
 
         then:
         failure.assertHasCause("Failed to apply plugin [id 'broken']")
-        failure.assertHasCause("throwing plugin")
+        failure.assertHasCause("java.lang.Exception: throwing plugin")
```

Comment:
Actually I made a mistake. This changed, because the exception is now wrapped by the build operation executor here: https://github.com/gradle/gradle/blob/4a2d701138519b88bf88a549c174be0a837b6be1/subprojects/core/src/main/java/org/gradle/internal/progress/DefaultBuildOperationExecutor.java#L130-L130